### ISSUE TITLE
Build: Image gets overlapped in mobile for portrait and landscape views

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#68]
URL: https://hcdev.openplus.ca/fr/node/2525/build#Section1 (click on Build for any Recall or Alert from Dashboard)
User Agent: Safari on iPhone 12
Language: French

Issue Description: The image for French gets overlapped by Modifier button in French pages in portrait and landscape views.

Steps to reproduce: View page on iPhone 12.

Expected results: Content should not overlap and should be visible all the time. The purpose of this image is not very clear. Suggestion would be to remove it.

![image](https://user-images.githubusercontent.com/14280062/129097251-cbd4e4f7-173b-4bed-a5ed-22b04a9cd545.png)

![image](https://user-images.githubusercontent.com/14280062/129097216-57997cdc-5df4-4a42-9151-7252a74fce1c.png)

**Comment by davidgotlieb** [comment]
Button moved over in smaller screens to no longer overlap

**Comment by ndolter** [comment]
Re-tested using an iPhone 11 (ios 15.4) and iPhone 12 in both portrait and landscape mode and the issue is now resolved.
